packages/common/src/lib/format-address.js

```diff
@@ -22,6 +22,25 @@ const formatAddress = (appealCaseData) => {
 	return addressComponents.filter(Boolean).join(', ');
 };
 
+/**
+ * @param {AppealCaseWithAppellant | AppealSubmission} appealCaseData
+ * @returns {string}
+ */
+const formatAddressWithBreaks = (appealCaseData) => {
```

Comment:
You could have made the joiner an optional argument of the existing `formatAddress` function then just pass in `'<br>'` when you need it ;)

Reply:
Changed!